elastic/kibana
# [ILM] Fix hiding/disabling searchable snapshot field when rollover is disabled

## Summary

Fix https://github.com/elastic/kibana/issues/85167

## How to test

1. Start Kibana and ES (with an enterprise license)
2. Create a new ILM policy
3. Disable rollover
4. Confirm that Searchable snapshot is not in Hot phase advanced settings and is disabled in cold phase actions.


Text in blue callout requires copy review.

## Screenshot

<img width="1082" alt="Screenshot 2020-12-07 at 19 05 56" src="https://user-images.githubusercontent.com/8155004/101387811-473cad80-38bf-11eb-9265-2a7721acf471.png">


### Checklist

Delete any items that are not applicable to this PR.
